src/gatsby-theme-conference/components/speakers.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import { Styled } from 'theme-ui'
+import { Box, Container } from 'theme-ui/layout'
+import SectionHeading from '../../../node_modules/gatsby-theme-conference/src/components/section-heading'
```

Comment:
You can reference this as you would other node modules with out the relative path